config/sprites/topographic/building_pnt_church.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><defs><style>.cls-1{fill:#fff;stroke:#231f20;stroke-miterlimit:10;stroke-width:0.8px;}.cls-2{fill:#231f20;}</style></defs><circle class="cls-1" cx="8.08" cy="8" r="7.41"/><polygon class="cls-2" points="7 4.44 7 1.43 9.03 1.43 9.03 4.44 12.07 4.44 12.07 6.51 9.04 6.51 9.04 14.57 7 14.57 7 6.52 3.93 6.52 3.93 4.46 7 4.44"/></svg>
```

Comment:
I think all these `id` and `data-name` need to be more specific instead of all as `Layer_1`.

Reply:
why do you think the id matters? i thought it was all done by file name?

Reply:
Just personal don't like the `Layer_1` things. :smile:. I think all the previous id are call `Sprites` which looks much more acceptable.

Reply:
these are all made with adobe or similar im not sure how much control you get for these ids maybe its based off the layr name in adobe?

Reply:
I think, abobe software have multiple layers when you creating graphs. And you can define the name as what you want. 
![image](https://user-images.githubusercontent.com/12163920/156672298-895680eb-f99c-4b3f-90ac-c5d67452ae4c.png)